collection-roller/src/main/scala/io/phdata/pulse/collectionroller/CollectionRollerMain.scala

```diff
@@ -79,6 +81,28 @@ object CollectionRollerMain extends LazyLogging {
     val (solr, solrService, collectionRoller) = createServices(parsedArgs)
     try {
       collectionRoller.collectionList().foreach(println)
+
+    } finally {
+      solrService.close()
+      solr.shutdown()
+    }
+    logger.info("Ending Collection Roller List App")
+  }
+
+  private def listApplicationsVerbose(parsedArgs: CollectionRollerCliArgsParser): Unit = {
+    logger.info("Starting Collection Roller List App")
+
+    val (solr, solrService, collectionRoller) = createServices(parsedArgs)
+    try {
+
+      println("Applications:")
+      collectionRoller.collectionList().foreach(println)
+      println("Aliases/Collections:")
+
+      for ((k, v) <- collection.mutable.LinkedHashMap(
+             collectionRoller.aliasMap().toSeq.sortBy(_._1): _*))
+        printf(" Alias : %s  Collection(s) : %s\n", k, v.mkString(","))
```

Comment:
Use ore descriptive names than 'k, v'. It's obvious what they are reading down, but they can easily be 'alias', 'collections'.

Using string interpolation can be more readable, try `s"Alias: $alias, Collections: ${collections.mkString(", ")"`

Reply:
What do you think about an output like this? It shows the same information as the non verbose output. 
```
application
    alias -> collections

````

```
asdf
    asdf_all  -> coll1, coll2
    asdf_latest -> coll3, coll4
fsda
    fsda_all -> coll1, coll2
    fsda_latest -> coll3, coll4
```

As a part of another later story it might be good to add the actual human readable dates onto the collections